src/components/CryptCard.tsx

```diff
@@ -0,0 +1,137 @@
+import {StyleSheet, Text, View, Image} from 'react-native';
+import React from 'react';
+import crypto from '../../metrics.json';
+import GainArrow from '../../assets/icons/north_east_black_24dp.svg';
+import LossArrow from '../../assets/icons/south_west_black_24dp.svg';
+
+type Props = {};
+
+const CryptCard = (Props: Props) => {
+  let percentageChange = (percentNumber: number) => {
+    let roundNumber = Math.round((percentNumber + Number.EPSILON) * 100) / 100;
+
+    if (roundNumber > 0) {
+      return (
+        <View style={styles.percentChangeText}>
+          <GainArrow width={16} height={16} fill="green" />
+          <Text style={styles.textGain}>{roundNumber + '%'}</Text>
+        </View>
+      );
+    }
+
+    if (roundNumber < 0) {
+      let roundNumberAbsolute = -roundNumber;
+      return (
+        <View style={styles.percentChangeText}>
+          <LossArrow width={16} height={16} fill="red" />
+          <Text style={styles.textLoss}>{roundNumberAbsolute + '%'}</Text>
+        </View>
+      );
+    }
+  };
+
+  return (
+    <View key={crypto.data.id} style={styles.container}>
+      <View style={[styles.col, styles.leftCol]}>
+        <Image
+          style={styles.image}
+          source={require('../../assets/image.png')}></Image>
+        <View style={[styles.colsText, styles.leftColText]}>
+          <Text style={styles.text}>{crypto.data.name}</Text>
+          <Text style={styles.text}>{crypto.data.symbol}</Text>
+        </View>
+      </View>
+      <View style={[styles.col, styles.rightCol]}>
+        <View style={[styles.colsText, styles.rightColText]}>
+          <Text style={styles.text}>
+            $
+            {Math.round(
+              (crypto.data.market_data.price_usd + Number.EPSILON) * 100,
+            ) / 100}
+          </Text>
+          {percentageChange(
+            crypto.data.market_data.percent_change_usd_last_24_hours,
+          )}
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default CryptCard;
+
+const styles = StyleSheet.create({
```

Comment:
styles should be stored in a separate file, the most common way to do this is with this format:

```
src/
   Components/
      AddButton/
          index.tsx <---- this file 
          styles.ts <---- move the styles here
```